### ISSUE TITLE
setup.py broken and README.md outdated

### ISSUE THREAD

**Issue by cm8** [kiwix/kiwix-build#188]
Since presumably a commit from March 2018 that renamed/refactored kiwix-build.py to __init__.py and introduced __main__.py, the documentation in README.md does not reflect current state of the package anymore.

In particular, running `pip3 install .` fails to create a console script.

I was able to work around the problem doing `python3 -m kiwixbuild` instead of running `kiwix-build` after changing into the cloned directory.  However, I'd think that a proper fix is to make setup.py create the proper console script.

**Comment by kelson42** [comment]
@cm8 Would you be able to propose a PR for these problems?

**Comment by mgautierfr** [comment]
Yes please @cm8, the patch is not related to the issue but it is very valuable, especially the part making `libzim/test/cluster.cpp` to use less memory.
